README.md

```diff
@@ -43,21 +43,53 @@ UCC is BSD-style licensed, as found in the [LICENSE](LICENSE) file.
 ### Developer's Build 
 ```sh
 $./autogen.sh
-
 $./configure --prefix=<ucc-install-path> --with-ucx=<ucx-install-path>
-
 $make 
 ```
 
 ### Build Documentation 
 ```sh
 $./autogen.sh
-
 $./configure --prefix=<ucc-install-path> --with-docs-only
-
 $make docs
 ```
 
+### Open MPI and UCC collectives
+
+#### Compile UCX 
```

Comment:
is it intentional to have smaller size header for compile section vs run section?

Reply:
It was, but it doesn't make sense.